packages/forms/src/form_builder.ts

```diff
@@ -315,13 +315,14 @@ export class FormBuilder {
  *
  * @publicApi
  */
-export interface NonNullableFormBuilder {
+@Injectable({providedIn: ReactiveFormsModule, useFactory: () => inject(FormBuilder).nonNullable})
```

Comment:
nit: just for readability, could you plz add a comma after the `useFactory` - that'd force a formatting tool to update the code to look like this:
```suggestion
@Injectable({
  providedIn: ReactiveFormsModule,
  useFactory: () => inject(FormBuilder).nonNullable,
})
```